apps/service-portal/src/components/Sidebar/ModuleNavigation.tsx

```diff
@@ -110,7 +116,7 @@ const ModuleNavigation: FC<Props> = ({
       {!collapsed && navArray && nav.enabled !== false && (
         <AnimateHeight
           duration={300}
-          height={isModuleActive || alwaysExpanded ? 'auto' : 0}
+          height={expand ? 'auto' : alwaysExpanded ? 'auto' : 0}
```

Comment:
Everything related to `alwaysExpanded` can be removed now with the new design.
It was only meant to be a temporary solution 🚀